libs/api/domains/social-insurance/src/lib/resolvers/general.resolver.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  CurrentUser,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+  type User,
+} from '@island.is/auth-nest-tools'
+import { Audit } from '@island.is/nest/audit'
+import { FeatureFlagGuard } from '@island.is/nest/feature-flags'
+
+import { ApiScope } from '@island.is/auth/scopes'
+import { UseGuards } from '@nestjs/common'
+import { Query, Resolver } from '@nestjs/graphql'
+import { UnionModel } from '../models/general/union.model'
+import { SocialInsuranceService } from '../socialInsurance.service'
+
+@Resolver()
+@UseGuards(IdsUserGuard, ScopesGuard, FeatureFlagGuard)
+@Audit({ namespace: '@island.is/api/social-insurance' })
+@Scopes(ApiScope.internal)
+export class GeneralResolver {
+  constructor(private readonly service: SocialInsuranceService) {}
+
+  @Query(() => [UnionModel])
+  async siaGetUnions(@CurrentUser() user: User) {
+    return this.service.getUnions(user)
+  }
+}
```

Comment:
Same as below,
The query name should reflect the object being fetched
https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions
```suggestion
@Resolver()
@UseGuards(IdsUserGuard, ScopesGuard, FeatureFlagGuard)
@Audit({ namespace: '@island.is/api/social-insurance' })
@Scopes(ApiScope.internal)
export class GeneralResolver {
  constructor(private readonly service: SocialInsuranceService) {}

  @Query(() => [UnionModel], {name: 'socialInsuranceUnions')})
  async siaGetUnions(@CurrentUser() user: User) {
    return this.service.getUnions(user)
  }
}
```

Reply:
Fixed. Added name.